Tribler/community/multichain/StatisticsDisplay/dbDriver.py

```diff
@@ -0,0 +1,191 @@
+import sqlite3
+
+default_database = ":memory:"
+
+list_neighbor_query = """
```

Comment:
Is it necessary to store all these statements as strings? This code looks so ugly. It is possible to move this to a separate file? Please don't mix dummy data with functionality.